lib/node_modules/@stdlib/math/stats/anova1/README.md

```diff
@@ -0,0 +1,164 @@
+          Treatment C: { mean: 7, sample_size: 3, id_number: 2, SD: 4 },
+          Control: { mean: 8, sample_size: 3, id_number: 3, SD: 4 } },
+      print: [Function: print],
+      method: 'One-Way anova1' }
+*/
+```
+
+The returned object comes with a `.print()` method which when invoked will print a formatted output of the results of the hypothesis test.
+
+```javascript
+var out;
+var x;
+var y;
+
+x = [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12 ];
+y = [ 'Treatent A', 'Treatment B', 'Treatment C', 'Control', 'Treatment A', 'Treatment B', 'Treatment C', 'Control', 'Treatment A', 'Treatment B', 'Treatment C', 'Control' ];
+
+// Making the object
+out = anova1(x, y);
+console.log( out.print() );
+/* =>
+One-Way anova1
+
+Null Hypothesis: All Means Equal
+Alternate Hypothesis: At Least one Mean not Equal
+
+                df    Sum Squares    Mean Sum Sq.       F Score     P Value
+Treatment       11    15                5               0.3125      0.8161
+Errors          8    128                16
+
+*/
+```
+
+The function accepts the following `options`:
+
+-   **alpha**: `number` in the interval `[0,1]` giving the significance level of the hypothesis test. Default: `0.05`.
+-   **decision**: a `boolean` value indicating if function is to return a decision of either _rejection of the null hypothesis_ or _failure to reject the null hypothesis_. Default: `false`
+
+By default, the test is carried out at a significance level of `0.05`. To choose a custom significance level, set the `alpha` option.
+
+```javascript
+var out;
+var x;
+var y;
+
+x = [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12 ];
+y = [ 'Treatent A', 'Treatment B', 'Treatment C', 'Control', 'Treatment A', 'Treatment B', 'Treatment C', 'Control', 'Treatment A', 'Treatment B', 'Treatment C', 'Control' ];
+
+// Making the object
+out = anova1(x, y);
+var table = out.print();
+/* e.g., returns
+ =>
+One-Way anova1
+
+Null Hypothesis: All Means Equal
+Alternate Hypothesis: At Least one Mean not Equal
+
+                df    Sum Squares    Mean Sum Sq.       F Score     P Value
+Treatment       11    15                5               0.3125      0.8161
+Errors          8    128                16
+
+*/
+
+out = anova1( x, y, {
+    'alpha': 0.01
+});
+table = out.print();
+/* e.g., returns
+=>
+One-Way anova1
+
+Null Hypothesis: All Means Equal
+Alternate Hypothesis: At Least one Mean not Equal
+
+df    Sum Squares    Mean Sum Sq.       F Score     P Value
+Treatment       11    15                5               0.3125      0.8161
+Errors          8    128                16
+
+Decision: Fail to reject null at alpha = 0.01
+*/
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   The calculation for the p value is based on an F distribution as indicated on [this website][penn-state]. 
+
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
```

Comment:
The example code in the `README.md` file is missing right now: It should match the example in `examples/index.js`.

Reply:
Resolved.